spec/factories/configuration_script_source.rb

```diff
@@ -9,5 +9,7 @@
 
   factory :embedded_ansible_configuration_script_source,
           :parent => :configuration_script_source,
-          :class  => "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource"
+          :class  => "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource" do
+    scm_url { "https://example.com/foo.git" }
```

Comment:
This PR effectively makes scm_url a required attribute for a ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource, which I think is ok.  Without this line you will consistently get a validation error on a GitRepository#url format.